src/OutputCollector.jl

```diff
@@ -264,8 +264,9 @@ function tail(collector::OutputCollector; len::Int = 100, colored::Bool = false)
     for line_idx in 1:len
         # We can run into UnicodeError's here
         try
-            idx = findprev(out, '\n', idx-1)
-            if idx <= 0
+            idx = findprev(equalto('\n'), out, idx-1)
+            if idx === nothing
```

Comment:
`findprev` still returns 0 for "not found" even with Compat on 0.6, so I'd make this `idx === nothing || idx == 0` (since you know in this case that the thing you're indexing is 1-based).

Reply:
somehow this still worked on 0.6?

Reply:
I guess because something was found? The difference is only in the case of something not being found.

Reply:
Oh.  Oh I see.  The `break` is just an early-exit.  It's just got bad performance by never going into the `if` statement.  :P